### PR TITLE
Add to docs with example of validation errors in templates

### PR DESCRIPTION
**Pull request by sampart**
I had to look at http://flask.pocoo.org/docs/0.12/patterns/wtforms/#forms-in-templates to see how to do this, and thought it would make a good addition to the Flask-WTF docs themselves.

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -63,4 +63,18 @@ Note that you don't have to pass ``request.form`` to Flask-WTF; it will
...
 if it is a POST request and if it is valid.
 
+If your forms include validation, you'll need to add to your template to display
+any error messages.  Using the ``form.name`` field from the example above, that
+would look like this:
+
+.. sourcecode:: html+jinja
+
+    {% if form.name.errors %}
+        <ul class=errors>
```
Comment (lepture) [inline docs/quickstart.rst]: Use proper HTML instead.

```
<ul class="errors">
```

Comment (davidism) [reply docs/quickstart.rst]: I'm fine with either. Armin is adamant that we don't use quotes or closing tags if we don't have to in the Flask docs.

Comment (sampart) [reply docs/quickstart.rst]: Whoops, sorry for the typo.  I've amended the commit and force-pushed, so it should be sorted now.  Thanks

**Comment by sampart** [comment]
Thank you!

**Pull request merged**